src/main/java/library/Template.java

```diff
@@ -0,0 +1,106 @@
+package library;
+
+import com.github.javafaker.Faker;
+import library.common.Common;
+import library.enumeration.BookAuthor;
+import library.enumeration.BookGenre;
+import library.enumeration.BookLanguage;
+
+import java.util.*;
+
+public class Template {
+
+    private List<BookItem> booksList;
+    List<BookItem> resultList;
```

Comment:
it is better to specify access modifier explicitly rather then use default all the time

Reply:
fixed